mautic/mautic
# More options for setup notifications

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/134 
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

#### Description:
PR added 2 new options to configuration pane:
1, 
OneSignal use gcm_sender_id as verification for web push service. 
They recommend use default value (482941778795), but in past we had to generate own gcm_sender_id  for every account.
It's better use it in configuration due backwards compatibility.
More in documentation, part. 2.2 Edit manifest.json:
https://documentation.onesignal.com/docs/web-push-sdk-setup-https
At the moment use Mautic gcm_sender_id  in code.

2, Options for disable welcome notification

![notifications](https://cloud.githubusercontent.com/assets/462477/20137624/dea049dc-a67b-11e6-8c39-b8ebb20f02cb.PNG)

#### Steps to test this PR:
1. Open http://your-mautic-domain.tld/manifest.json before apply PR
2. Apply PR, go to Configuration > Web Notification settings and enable  it
3. Change default Shared key for push notifications from documentation to your anyhting else
4. See change in url http://your-mautic-domain.tld/manifest.json
5. Disable Welcome Notification
6. Open whatever landing page and see source code, you should see something like this
![notification_example_code](https://cloud.githubusercontent.com/assets/462477/20138043/5c446e66-a67e-11e6-8493-9de469564736.PNG)
